modules/UI.js

```diff
@@ -0,0 +1,34 @@
+import Store from './store.js';
+
+const store = new Store();
+
+export default class UI {
+  static displayBooks() {
+    const books = store.getBooks();
+    books.forEach((book) => UI.addBookList(book));
+  }
+
+  static addBookList(book) {
+    const bookList = document.getElementById('book-list');
+
+    const content = document.createElement('div');
+    content.innerHTML = `
+      <div>${book.title} By ${book.author}</div>
+      <button id="book-num-${book.id}"class="delete">Remove</button>
+      `;
+
+    bookList.appendChild(content);
+    content.classList.add('book-row-content');
+  }
+
+  static deleteBook(element) {
+    if (element.classList.contains('delete')) {
+      element.parentElement.remove();
+    }
+  }
+
+  static clearFields() {
+    document.querySelector('#title').value = '';
+    document.querySelector('#author').value = '';
+  }
+}
```

Comment:
- [ ] Good job so far your code looks clean and awesome 👏 However as part of the requirements we should stick with ES6 so we need always to use arrow functions, I noticed that all of your classes are normal functions, and not arrow functions, example : 

```JS
  // Normal Function
   static addBookList(book) { ... }
  // Arrow Function
    addBookList= (book) => { ... }
```

kindly fix all of your normal functions, including the ones in `list-book.js`,`store.js`, `showBook.js` 🙏